microsoft-edge/webview2/release-notes/index.md

```diff
@@ -67,11 +67,344 @@ See also:
+   * [CoreWebView2Notification.Language Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#language)
+   * [CoreWebView2Notification.RequiresInteraction Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#requiresinteraction)
+   * [CoreWebView2Notification.ShouldRenotify Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#shouldrenotify)
+   * [CoreWebView2Notification.Tag Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#tag)
+   * [CoreWebView2Notification.Timestamp Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#timestamp)
+   * [CoreWebView2Notification.Title Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#title)
+   * [CoreWebView2Notification.VibrationPattern Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#vibrationpattern)
+<!-- Methods -->
+   * [CoreWebView2Notification.ReportClicked Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#reportclicked)
+   * [CoreWebView2Notification.ReportClosed Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#reportclosed)
+   * [CoreWebView2Notification.ReportShown Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#reportshown)
+<!-- Events -->
+   * [CoreWebView2Notification.CloseRequested Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notification?view=webview2-winrt-1.0.2739.15&preserve-view=true#closerequested)
+
+* [CoreWebView2NotificationReceivedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notificationreceivedeventargs?view=webview2-winrt-1.0.2739.15&preserve-view=true)
+   * [CoreWebView2NotificationReceivedEventArgs.Cancel Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notificationreceivedeventargs?view=webview2-winrt-1.0.2739.15&preserve-view=true#cancel)
```

Comment:
should the 3 tabs have `CoreWebView2NotificationReceivedEventArgs.Cancel`, or not?  
@peiche-jessica @nishitha-burman -  The research below indicates that I should delete the 1 Cancel that came in, here.  it came in in the orig [commit](https://github.com/MicrosoftDocs/edge-developer/pull/3250/commits/7129e6e96f36d1503f41b2b086f371d5ffbe5588)

it's currently inconsistent:
.NET tab: lacks Cancel.
WinRT tab: has Cancel.
Win32 tab: lacks Cancel

.NET Ref: doesn't exist yet; instead, check live latest (promoted to Stable in Prerelease): 
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2notificationreceivedeventargs?view=webview2-dotnet-1.0.2730-prerelease - lacks Cancel.
WinRT Ref: check open PR:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2notificationreceivedeventargs?view=webview2-winrt-preview&branch=pr-en-us-64 - lacks Cancel.
Win32 Ref: check open PR:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs?view=webview2-win32-preview&branch=pr-en-us-97 - lacks Cancel

we're in the section "promoted to Stable in Release", therefore earlier RelNotes section exists, and earlier Ref exists: 
* promoted to Stable in Prerelease.
* Experimental in Prerelease.

rendered article section: 2nd tab set:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3250#promotions-1

there are no other mentions of
CoreWebView2NotificationReceivedEventArgs.Cancel
in relnotes file.  only this incoming mention.

Reply:
Please remove accordingly. Thanks.

Reply:
removed CoreWebView2NotificationReceivedEventArgs.Cancel